app/budget/budgetCtrl.js

```diff
@@ -5,6 +5,8 @@ budgetApp.controller('BudgetCtrl', ['$scope', '$rootScope', '$window', '$locatio
 
 		$scope.view = {};
 
+		$scope.budgetConfigStyles = { "width" : "40%" };
```

Comment:
Is there a reason this isn't in CSS?

Reply:
The variable budgetConfigStyles will be handed to the generic ipa-modal component and used in an ng-style.